cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/AbstractMapReduceContextBuilder.java

```diff
@@ -63,6 +63,7 @@ public BasicMapReduceContext build(MapReduceMetrics.TaskType type,
                                      String taskId,
                                      long logicalStartTime,
                                      String workflowBatch,
+                                     String adapterName,
```

Comment:
Fixed. Wanted to keep it next to ``workflowBatch``, but don't feel too strongly about it.